server/plugin/disconnect.go

```diff
@@ -6,7 +6,12 @@ import (
 )
 
 func (p *Plugin) handleDisconnect(c *plugin.Context, args *model.CommandArgs, _ []string) string {
-	err := p.client.KV.Delete(args.UserId + "_token")
+	var encryptedToken []byte
+	_ = p.client.KV.Get(getUserTokenKey(args.UserId), &encryptedToken)
```

Comment:
What if there's a problem retrieving the item from the KVStore?

Reply:
This check is for checking if user is connected and then send appropriate message. if the get call fails for some reason then i think it's okay to let user proceed to actual disconnect operation and if there is any underlying issue with the server then delete operation will error out which is handled by sending appropriate message to user.